README.md

```diff
@@ -15,7 +15,138 @@ def deps do
 end
 ```
 
+## Configuration
+
+Provide your `TAG_ID` in your configuration file:
```

Comment:
Provide a ... looks better to avoid repeating `your`

Reply:
Fixed.